torch/_torch_docs.py

```diff
@@ -5156,11 +5165,22 @@ def merge_dicts(*dicts):
             [ 0.9206]]])
     >>> torch.quantile(a, q, dim=1, keepdim=True).shape
     torch.Size([3, 2, 1])
+    >>> a = torch.arange(6.)
+    >>> a
+    tensor([0., 1., 2., 3., 4., 5.])
```

Comment:
Great example. I would add the linear interpolation method and change the q value to 0.6 so there is a difference between linear and midpoint.

Reply:
Thank you so much for pointing this out.
The linear interpolation method is added to the example, and the `q` value is changed to 0.6 to demonstrate the difference between `linear` and `midpoint`.